client/src/components/GetDetails/index.js

```diff
@@ -1,21 +1,29 @@
-import React from 'react';
+import React, { Component } from 'react';
 import Title from '../Shared/Title';
 import Header from '../Shared/Header';
 import Form from '../Shared/Form';
 import GButton from '../Shared/GreenButton';
 import Button from '../Shared/Button';
 import Footer from '../Shared/Footer';
 
-function GetDetails() {
-  return (
-    <React.Fragment>
-      <Title />
-      <Header />
-      <Form />
-      <Button />
-      <GButton />
-      <Footer />
-    </React.Fragment>
-  );
+class GetDetails extends Component {
```

Comment:
Same as the other file, if you don't have state you can use a functional stateless component.

Reply:
but we will have state later?